### PR TITLE
[kube-prometheus-stack] - Add support for templating in additionalAle…

### PR DESCRIPTION
**Pull request by Allex1**
…rtManagerConfigs

Signed-off-by: birca <Allex21ro@yahoo.com>

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by Allex1** [comment]
@monotek @andrewgkew @desaintmartin @gianrubio @gkarthiks @scottrigby @Xtigyro please have a look

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.11.0
+version: 39.11.1
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:25]:
```suggestion
version: 39.13.1
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (Allex1) [reply charts/kube-prometheus-stack/Chart.yaml]: done

File: charts/kube-prometheus-stack/Chart.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.13.0
+version: 39.13.1
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:25]:
```suggestion
version: 39.13.2
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 2 above)
Comment (Allex1) [reply charts/kube-prometheus-stack/Chart.yaml]: @monotek done

**Review by monotek — APPROVED** [review]

**Pull request merged**